pkg/kubelet/server/stats/handler.go

```diff
@@ -215,13 +216,16 @@ func (h *handler) handleStats(request *restful.Request, response *restful.Respon
 // If "only_cpu_and_memory" GET param is true then only cpu and memory is returned in response.
 func (h *handler) handleSummary(request *restful.Request, response *restful.Response) {
 	onlyCPUAndMemory := false
-	request.Request.ParseForm()
+	err := request.Request.ParseForm()
+	if err != nil {
+		handleError(response, "/stats/summary", errors.Wrapf(err, "parse form failed"))
```

Comment:
Can you talk more about why we are appending the `parse form failed` component? Will that not be apparent from the `err`?

Reply:
It helps us better understand the context where the error is produced and It allows me to check in the test that effectively the error was produced when parsing the form.

Reply:
Mhhm interesting! Curious if others have strong opinions - my personal vote would probably be that it isn't necessary, but I'm flexible.